commons-geometry-core/src/main/java/org/apache/commons/geometry/core/Vector.java

```diff
@@ -93,11 +93,22 @@
 
     /** Get a normalized vector aligned with the instance. The returned
      * vector has a magnitude of 1.
-     * @return a new normalized vector
-     * @throws IllegalArgumentException if the norm is zero, NaN, or infinite
+     * @return normalized vector
+     * @throws IllegalArgumentException if the norm (or norm inverse, if applicable)
+     *      is zero, NaN, or infinite
```

Comment:
This space is correct?

Reply:
I find it easier to read when the second line of tag content is indented. It doesn't affect the generated docs so it's a personal preference thing. Is that what you're referring to?